collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/FormEntry.java

```diff
@@ -123,7 +123,7 @@ public static void clickOk() {
     }
 
     public static void clickOptionsIcon() {
-        onView(withContentDescription("More options")).perform(click());
+        Espresso.openContextualActionModeOverflowMenu();
```

Comment:
Should this be `openActionBarOverflowOrOptionsMenu` instead? Looking at the [docs](https://developer.android.com/reference/androidx/test/espresso/Espresso.html#openContextualActionModeOverflowMenu()) I'm not 100% sure as is it's a little confusing. It seems `openContextualActionModeOverflowMenu` would also work on "floating ActionModes" (I'm assuming like the context menus on long presses on text).

Reply:
Yeah, both work well and I'm also not 100% sure but openActionBarOverflowOrOptionsMenu seems like a better method here so fixed.